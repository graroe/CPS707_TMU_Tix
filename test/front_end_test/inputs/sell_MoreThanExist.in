login
sales
sell
CirqueDuSoleil
8
sell
CirqueDuSoleil
3
2
sell
CirqueDuSoleil
!q
logout