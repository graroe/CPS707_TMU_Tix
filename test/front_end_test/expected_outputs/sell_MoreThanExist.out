Enter session type <sales or admin>: 
Sales login successful.
Enter event name:
Current number of tickets: 10\nEnter number of tickets to sell:
8 tickets sold. New total: 2
Enter event name:
Current number of tickets: 2\nEnter number of tickets to sell:
Maximum exceeded.\nCurrent number of tickets: 2\nEnter number of tickets to sell: 
2 tickets sold. New total: 0
Enter event name:
That event is sold out.\nEnter event name: 
Transaction canceled
Session terminated.